examples/cis5-smart-contract-wallet/src/lib.rs

```diff
@@ -2,43 +2,44 @@
+//! CCD/CIS-2 tokens can be deposited into the smart contract wallet
 //! by specifying to which public key ([PublicKeyEd25519] schema) the deposit
-//! should be assigned. Authorization for token and currency transfers from the
+//! should be assigned. Authorization for token and CCD transfers from the
 //! smart contract's assigned public key balance is exclusively granted to the
 //! holder of the corresponding private key, ensuring self-custodial control
 //! over the assets.
 //!
-//! Transfers of native currency and CIS-2 token balances (meaning `withdraw`
-//! and `internalTransfer` functions) do not require
-//! on-chain transaction submissions. Instead, the holder of the corresponding
-//! private key can generate a valid signature and identify a third party to
-//! submit the transaction on-chain, potentially incentivizing the third-party
-//! involvement through a service fee. The message that was signed specifies the
-//! amount of service fee and the service fee recipient's public key.
+//! Transfers of CCD and CIS-2 token balances (meaning the `withdraw` and
+//! `transfer` functions) do not require on-chain transaction submissions.
+//! Instead, the holder of the corresponding private key can generate a valid
+//! signature and identify a third party to submit the transaction on-chain,
+//! potentially incentivizing the third-party involvement through a service fee.
+//! The message that was signed specifies the amount of service fee and the
+//! service fee recipient's public key.
 //!
-//! Any withdrawal (native currency or CIS-2 tokens) to a smart contract will
+//! Any withdrawal (CCD or CIS-2 tokens) to a smart contract will
 //! invoke a `receiveHook` function on that smart contract.
 //!
 //! The three main actions in the smart contract wallet that can be taken are:
 //! - *deposit*: assigns the balance to a public key within the smart contract
 //!   wallet.
-//! - *internal transfer*: assigns the balance to a new public key within the
-//!   smart contract wallet.
+//! - *transfer*: assigns the balance to a new public key within the smart
+//!   contract wallet.
 //! - *withdraw*: withdraws the balance out of the smart contract wallet to a
 //!   native account or smart contract.
 //!
 //! The goal of this standard is to simplify the account creation onboarding
-//! flow on Concordium. Users can hold/transfer native currency/CIS-2 tokens
+//! flow on Concordium. Users can hold/transfer CCD/CIS-2 tokens
 //! without a native account as a starting point (no KYC required). Once users
 //! are ready to go through the KYC process to create a native account on
 //! Concordium, they can withdraw assets out of the smart contract wallet.
 //! The public key accounts in this smart contract wallet can't submit the
 //! transactions on chain themselves, but rely on someone with a native account
 //! (third-party) to do so.
+use concordium_cis2 as cis2;
 use concordium_cis2::*;
```

Comment:
```
use concordium_cis2::{self as cis2, *};
```

Reply:
Side note: should this not be handled by the formatter?

Reply:
Not tested, but it could be that the formatter config file has to be in every example folder to be picked up correctly.
https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/rustfmt.toml

or we are missing this specific rule in the config file.

Reply:
I just checked, and it is definitely picking up the file. After comparing it with some of our other projects, the differences in `rustfmt.toml` are tiny and don't seem like they would influence imports. It seems like it is just not something that is done automatically.